zhaquirks/tuya/__init__.py

```diff
@@ -1065,7 +1065,7 @@ def __init__(self, *args, **kwargs):
         self.last_tsn = -1
         super().__init__(*args, **kwargs)
 
-    class ServerCommandDefs(OnOff.ServerCommandDefs):
+    class ClientCommandDefs(OnOff.ClientCommandDefs):
         """Server command definitions."""
```

Comment:
nit: we should also change the docstring if we adjust this
```suggestion
        """Client command definitions."""
```